.env.example

```diff
@@ -4,3 +4,7 @@ RUST_LOG=debug
 KAKAROT_HTTP_RPC_ADDRESS=0.0.0.0:3030
 KAKAROT_ADDRESS=0x7a88f6f9d63ccaa5855babb32cbb0230b8588aaaa6bc4ce2d173fa528ce7567
 PROXY_ACCOUNT_CLASS_HASH=0x3010a53967fa04842bcbcb6de8817101f047ef0d074b3eacbe714a3fc42a2eb
+
+# configurations for testing
+COMPILED_KAKAROT_PATH=crates/core/tests/compiled_solidity
+KAKAROT_EVM_PROJECT_DIR=
```

Comment:
`KAKAROT_EVM_PROJECT_DIR` what does this mean?
is this simply the folder where smart contracts live?

Could just call it `SOLIDITY_CONTRACTS` then to match the folder you committed

Reply:
its used by the `scripts/copy_kakarot_deploy_artifacts.sh` to build kakarot v0 contracts and move its deploy artifacts to where the tests expect them (the `COMPILED_KAKAROT_PATH`)

Is this sensible/clear? 👂